docs/getting-started/install-configure.md

```diff
@@ -1,20 +1,54 @@
 ---
-title: Install & Configure
-toc: true
-weight: 2
-indent: true
+title: Install & Configure toc: true weight: 2 indent: true
```

Comment:
Looks like this got wrapped unexpectedly. :(

Reply:
Apologies for these mistakes. My editor's autoformat for line width consolidated these lines. Pushing to fix.